File: Python/백준/Bronze/10250. ACM 호텔/ACM 호텔.py
q=int(input())
for i in range(q):
    a,b,c = map(int,input().split())
    f = c%a
    if(f==0):
        f=a
    h = (c-1)//a+1
    if(h<10):
        print(f"{f}0{h}")
    else:
        print(f"{f}{h}")

File: C#/백준/Bronze/10250. ACM 호텔/ACM 호텔.cs
using System;

namespace q{
    class w{
        static void Main(){
            int q = int.Parse(Console.ReadLine());
            for (int i =0; i<q; i++){
                string[] aa = Console.ReadLine().Split();
                int a = int.Parse(aa[0]);
                int b = int.Parse(aa[1]);
                int c = int.Parse(aa[2]);
                int f = c%a;
                if(f==0){
                    f=a;
                }
                int j = (c-1)/a+1;
                if(j<10){
                    Console.WriteLine($"{f}0{j}");
                }
                else{
                    Console.WriteLine($"{f}{j}");
                }
            }
        }
    }
}